### PR TITLE
Add index.html, styles.css and .gitignore file 

### PR DESCRIPTION
**Pull request by Maria18000**
Index.html styles.css and .gitignore files are added per the requirements.
ReadME.md file content was taken from the template and edited.

### CODE REVIEW

**Review by AmaduKamara — COMMENTED** [review]
Hi @Maria18000,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have Linters setup. Kindly set up Linters on your project. It's required you set up Linters as it helps in checking your code so you write clean and better code.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+node_modules/
+package-lock.json
+package.json
```
Comment (ShadyShawkat) [inline .gitignore]: - [x] [OPTIONAL] Please keep in mind that `package.json` and `package-lock.json` contain information on the dependency packages required to install to run your project. That means if I were to try to run your project, I wouldn't be able to do so, as I don't know what are the dependency packages required to install. 
- Kindly remove those two files from your `.gitignore` and make sure to push them to your repo.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @Maria18000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (Maria18000) [reply .gitignore]: Thank you! Both file names have been removed from .gitignore

**Pull request merged**